docsrc/content/codec.fsx

```diff
@@ -1,11 +1,10 @@
 (*** hide ***)
 // This block of code is omitted in the generated HTML documentation. Use
 // it to define helpers that you do not want to show in the documentation.
-#r @"../../src/Fleece.SystemJson/bin/Release/netstandard2.1/System.Json.dll"
```

Comment:
The code you replace is actually the compiled binary, while here you change it to be the released library version